### ISSUE TITLE
Inconsistent numbers

### ISSUE THREAD

**Issue by hueniverse** [delvedor/router-benchmark#3]
The order in which tests are run changes the numbers completely. What am I missing?

```
find-my-way | lookup static route x 6,363,699 ops/sec ±1.82% (90 runs sampled)
route-recognizer | lookup static route x 322,352 ops/sec ±1.05% (88 runs sampled)
router | lookup static route x 147,300 ops/sec ±11.72% (43 runs sampled)
server-router | lookup static route x 133,705 ops/sec ±2.91% (66 runs sampled)
koa-router | lookup static route x 246,221 ops/sec ±5.57% (55 runs sampled)
routr | lookup static route x 654,270 ops/sec ±5.24% (73 runs sampled)
call | lookup static route x 752,269 ops/sec ±3.34% (37 runs sampled)
express | lookup static route x 134,148 ops/sec ±37.03% (37 runs sampled)
barista | lookup static route x 125,760 ops/sec ±22.15% (41 runs sampled)
```

```
call | lookup static route x 3,392,290 ops/sec ±3.52% (91 runs sampled)
find-my-way | lookup static route x 6,261,991 ops/sec ±1.06% (87 runs sampled)
route-recognizer | lookup static route x 323,487 ops/sec ±1.36% (85 runs sampled)
router | lookup static route x 150,585 ops/sec ±12.50% (46 runs sampled)
server-router | lookup static route x 137,907 ops/sec ±2.63% (48 runs sampled)
koa-router | lookup static route x 243,716 ops/sec ±5.60% (56 runs sampled)
routr | lookup static route x 619,315 ops/sec ±3.67% (22 runs sampled)
express | lookup static route x 129,364 ops/sec ±42.50% (38 runs sampled)
barista | lookup static route x 177,160 ops/sec ±6.49% (48 runs sampled)
```

```
express | lookup static route x 151,606 ops/sec ±12.54% (47 runs sampled)
call | lookup static route x 814,631 ops/sec ±6.26% (45 runs sampled)
find-my-way | lookup static route x 1,446,100 ops/sec ±14.30% (31 runs sampled)
route-recognizer | lookup static route x 61,909 ops/sec ±5.95% (71 runs sampled)
router | lookup static route x 130,635 ops/sec ±44.19% (36 runs sampled)
server-router | lookup static route x 149,714 ops/sec ±3.00% (76 runs sampled)
koa-router | lookup static route x 256,921 ops/sec ±4.78% (60 runs sampled)
routr | lookup static route x 731,795 ops/sec ±4.55% (84 runs sampled)
barista | lookup static route x 169,081 ops/sec ±7.52% (53 runs sampled)
```

**Comment by delvedor** [comment]
That's a good question, I don't know.
I'll try to investigate :)

**Comment by delvedor** [comment]
Suggestions addressed in #7 and #8, thank you for contributing! :)
